docs/DeveloperGuide.adoc

```diff
@@ -464,6 +464,40 @@ We are using `java.util.logging` package for logging. The `LogsCenter` class is
 
 Certain properties of the application can be controlled (e.g App name, logging level) through the configuration file (default: `config.json`).
 
+// tag::unlockcommand[]
+=== Unlock Feature
+
+==== Introduction
+
+We have implemented an unlock command that focuses on the `Storage` component. An `UnlockCommand` class is added
+and an `UnlockEvent` event is also added.
+
+
+==== Current Implementation
+
+The `unlock` command allows changes made to be saved to the address book.
+
+Without the correct (for now hardcoded) password, all the changes made (add, edit, delete)during the session
+will not be saved and will not be shown once the window is closed and reopened.
+
+The following activity diagram shows how the current `Unlock` command works:
+
+image::UnlockActivityDiagram.png[width="800"]
```

Comment:
I think the picture is over-stretched. maybe u can change the width size smaller, to look at the actual doc, click view at files changed and to ur implementation

Reply:
as yours is in portrait, maybe u can try height="600" or height="650"